### DISCUSSION TITLE
How do I install this thing? lol

### DISCUSSION THREAD

**Discussion by mars-s** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#396] [Q&A]
i dont see a install guide like install.md for mac os, I have a MacBook Pro 16 2019 with Radeon pro 5500m

**Comment by bsavery** [comment]
https://radeon-pro.github.io/RadeonProRenderDocs/en/plugins/blender/installation.html

**Reply by mars-s** [reply]
i tried installing the mac os file but it just renames to this, not a zip file that i can add into blender
<img width="765" alt="Screen Shot 2021-11-20 at 7 31 01 pm" src="https://user-images.githubusercontent.com/71009876/142719874-5617756d-b868-42b9-8cc8-6c9782ada3cd.png">

**Comment by mars-s** [comment]
ok i managed to zip it up and get it working but now the rendering engine is crashing n giving me thi
<img width="923" alt="Screen Shot 2021-11-20 at 7 37 42 pm" src="https://user-images.githubusercontent.com/71009876/142720083-9c091e10-b26e-4196-9777-4a178b83758e.png">
s

**Reply by mars-s** [reply]
ok turns out the denoiser kept crashing it i just had to turn that off so now its working fine

**Comment by woop120** [comment]
![image](https://user-images.githubusercontent.com/95972277/145681424-78141b08-d740-4920-ba46-04edc982159d.png)

why do i get this error message when trying to enable the add-on

**Reply by bsavery** [reply]
You need to use the weekly build with blender 3.0
